src/components/subtitlesettings/subtitlesettings.template.html

```diff
@@ -34,6 +34,14 @@ <h2 class="sectionTitle">
         </div>
     </div>
 
+    <div class="checkboxContainer checkboxContainer-withDescription">
+        <label>
+            <input is="emby-checkbox" type="checkbox" id="chkAlwaysBurnInSubtitleWhenTranscoding" />
+            <span>${AlwaysBurnInSubtitleWhenTranscoding}</span>
```

Comment:
The localisation for `AlwaysBurnInSubtitleWhenTranscoding` and `AlwaysBurnInSubtitleWhenTranscodingHelp` was accidentally removed.